src/removeBatch.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import type { MutableState, Mutator, Tools } from 'final-form'
+
+const removeBatch: Mutator = (
+  [name, indexes]: any[],
+  state: MutableState,
+  { changeValue }: Tools
+) => {
+  changeValue(
+    state,
+    name,
+    (array: ?(any[])): ?(any[]) => {
```

Comment:
This could simply be:
`array => array && indexes ? array.filter((item, i) => !indexes.includes(i)) : array`

Although much simpler, the performance is significantly worse. Not sure what should be the goal here, since probably there won't be hundreds/thousands of item in the field array.

Any thoughts?